includes/utils.php

```diff
@@ -193,12 +193,16 @@ public static function replace_urls( $from, $to ) {
 
 		global $wpdb;
 
-		// @codingStandardsIgnoreStart cannot use `$wpdb->prepare` because it remove's the backslashes
 		$rows_affected = $wpdb->query(
-			"UPDATE {$wpdb->postmeta} " .
-			"SET `meta_value` = REPLACE(`meta_value`, '" . str_replace( '/', '\\\/', $from ) . "', '" . str_replace( '/', '\\\/', $to ) . "') " .
-			"WHERE `meta_key` = '_elementor_data' AND `meta_value` LIKE '[%' ;" ); // meta_value LIKE '[%' are json formatted
-		// @codingStandardsIgnoreEnd
+			$wpdb->prepare(
+				"UPDATE {$wpdb->postmeta} " .
+				'SET `meta_value` = REPLACE(`meta_value`, %s, %s) ' .
+				"WHERE `meta_key` = '_elementor_data' AND `meta_value` LIKE %s ;",
+				str_replace( '/', '\\/', $from ),
+				str_replace( '/', '\\/', $to ),
+				'[%' // meta_value LIKE '[%' are json formatted
+			)
+		);
```

Comment:
```suggestion
			$query = "UPDATE {$wpdb->postmeta} SET `meta_value` = REPLACE(`meta_value`, %s, %s) WHERE `meta_key` = '_elementor_data' AND `meta_value` LIKE '[%';"

			$wpdb->prepare( $query, $from, $to );
		);
```

I think it's more readable.
\+ I think there is no need for `str_replace` to the URLs if you are using `prepare` BUT please validate me.